lib/v2/finology/mostViewed.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable linebreak-style */
+
+const { error } = require('winston');
```

Comment:
Please use the exported logger instead of creating a new instance.
https://github.com/DIYgod/RSSHub/pull/12941#discussion_r1285912877